torchci/pages/api/flaky-tests/[testinfo].ts

```diff
@@ -0,0 +1,33 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import fetchFlakyTests from "lib/fetchFlakyTests";
+import { FlakyTestData } from "lib/types";
+
+interface Data {}
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  // Expect testinfo to be /testName%20testSuite%20testFile
+  const testinfo = req.query.testinfo;
```

Comment:
I think we should split this out into parameters like they do in `torchci/pages/api/[repoOwner]/[repoName]/commit/[sha].ts` instead of doing the split. Or make them query parameters like flaky-test?testName=blah&testSuite=blahblah&testFile=blahblahb

Reply:
oooh i didn't realize this was a thing we could do, def better